packages/zosjobs/src/GetJobs.ts

```diff
@@ -393,4 +381,19 @@ export class GetJobs {
         Logger.getAppLogger().info("GetJobs.getSpoolContentCommon() parameters: " + parameters);
         return ZosmfRestClient.getExpectString(session, JobsConstants.RESOURCE + parameters, [Headers.TEXT_PLAIN_UTF8]);
     }
+
+    private static filterResultsByStatuses(jobs: IJob[], params: IGetJobsParms): IJob[] {
```

Comment:
Should this be a public method ?

Reply:
The users of zowe-cli api are suppose to call getJobsByParameters and that will call zOSMF api and handle filtering results. filterResultsByStatuses is suppose to be an internal function. I separated filtering into a function for easier unit testing and more readable code.